### DISCUSSION TITLE
Max Text Size / API Limit

### DISCUSSION THREAD

**Discussion by RohanVDeshpande** [pndurette/gTTS#293] [General]
Are there any limits to how much text can be converted? Is there a cap on the number of API calls per unit time that can be made?

**Comment by pndurette** [comment]
There is but it's unknown. Since this is an endpoint that doesn't use authorization, the only known limit is based on how much Google tolerates your requests, i.e. your IP will get banned for a few hours if you hammer the endpoint. So rule of thumb is to be reasonable.
